content/blog/kubernetes-local.fr.md

```diff
@@ -0,0 +1,266 @@
+Kubernetes et son fonctionnement, on tombe rapidement sur des configurations
+matérielles ou virtuelle relativement complexes. Le cluster etcd peut être
+installé sur les mêmes nodes hébergeant l'api-server et les autres éléments du
+control plane. Le control plane peut être constitué de un ou plusieurs nodes en
+fonction du niveau de résilience souhaité. Pareil pour etcd. De multiples
+configurations sont possibles et chacune à ses avantages et ses inconvénients.
+Une fois que cette configuration est choisie, la seconde question que l'on se
+pose est bien souvent "où ?". En effet, l'endroit où sera déployé Kubernetes
+est un choix important puisque certaines fonctions (Service loadbalancer,
+Storage Class, etc) ne seront peut être pas disponibles.
+
+Lorsqu'on souhaite déployer Kubernetes afin de s'entrainer, expérimenter,
+tester Kubernetes, cela fait beaucoup de questions sans réponse.
+
+Nous vous proposons un aperçu des possibilités qui s'offrent à vous pour
+déployer Kubernetes en local sur votre poste de travail sans compromis sur les
+fonctionnalités.
+
+## La bonne recette
+
+Lorsqu'on souhaite travailler sur Kubernetes en local, on cherche tout d'abord
+la simplicité. On ne souhaite pas passer plusieurs heures à prendre en main des
+outils comme [Kubespray](https://kubespray.io) pour déployer Kubernetes.
+Particule a développé une version "plus légère" de Kubespray,
+[Symplegma](https://github.com/particuleio/symplegma) mais même ça c'est encore
+trop lourd. On souhaite quelque chose de simple à utiliser, qui puisse démarrer
+et supprimer un cluster Kubernetes en quelques minutes. On veut aussi avoir
+accès à l'ensemble de l'API, pas à un subset et à la possibilité d'utiliser
+toutes les fonctionnalités comme si on était sur un cluster de production.
+
+En terme d'architecture, c'est simple on veut généralement un seul node, qui
+fera office de control plane et de worker. On souhaiterai néanmoins avoir le
+choix de faire du multi-nodes si l'envie nous en prenait. Pour tester des
+fonctions de schéduling ou de communication inter-nodes, c'est plutôt utile.
+
+## Kubeadm : le projet officiel
+
+![kubeadm](https://raw.githubusercontent.com/kubernetes/kubeadm/master/logos/stacked/color/kubeadm-stacked-color.png#logosize)
+
+[Kubeadm](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/)
+ (à prononcer kube admin et non pas kubeADM) est le projet officiel
+de Kubernetes permettant de bootstraper un cluster. Kubeadm se compose d'un
+binaire incluant une ligne de commande assez complète. Kubeadm est utilisé par
+un nombre important de projet tiers comme la solution pour bootstraper un
+cluster Kubernetes. Mais c'est là sa limite, il ne fait que ça. En soit pour
+démarrer un control plane Kubernetes, il suffit de :
+
+```
+$ kubeadm init
+```
+
+Mais pour bénéficier en sortie d'un control plane fonctionnel, cela
+présupposeque plusieurs choses soient installées, en amont ou en aval :
```

Comment:
Minor typo in `présupposeque` (missing whitespace between `présuppose` and `que`)